apps/web/src/components/editor/action-bar.tsx

```diff
@@ -439,10 +439,27 @@ function Tab(props: TabProps) {
     : Note;
   const { attributes, listeners, setNodeRef, transform, transition, active } =
     useSortable({ id });
+  const activeTabRef = useRef<HTMLElement | null>(null);
+
+  useEffect(() => {
+    if (activeTabRef.current) {
+      const tab = activeTabRef.current;
+      tab.scrollIntoView({
+        behavior: "smooth",
+        block: "nearest",
+        inline: "nearest"
+      });
+    }
+  }, [isActive]);
 
   return (
     <Flex
-      ref={setNodeRef}
+      ref={(el) => {
+        setNodeRef(el);
+        if (isActive) {
+          activeTabRef.current = el;
+        }
```

Comment:
No need to wrap this in a condition.

Reply:
Without the condition, the tab strip would scroll to the last tab regardless of the active tab position.

Reply:
You are putting the condition in the wrong place then. It should be in the `useEffect`.